docs/blog/posts/2024/srlinux-asymmetric-routing.md

```diff
@@ -0,0 +1,1801 @@
+            default-action {
+                policy-result reject
+            }
+            statement bgp-underlay {
+                match {
+                    protocol bgp
+                    family [
+                        ipv4-unicast
+                        ipv6-unicast
+                    ]
+                }
+                action {
+                    policy-result accept
+                }
+            }
+            statement bgp-evpn-overlay {
+                match {
+                    family [
+                        evpn
+                    ]
+                }
+                action {
+                    policy-result accept
+                }
+            }
+        }
+    }
+--{ + running }--[  ]--
+```
+
+///
+
+/// tab | spine1
+
+```{.srl .code-scroll-lg}
+--{ running }--[  ]--
+A:spine1# info routing-policy policy leaf-*
+    routing-policy {
+        policy leaf-export {
+            default-action {
+                policy-result reject
+            }
+            statement loopback {
+                match {
+                    protocol local
+                }
+                action {
+                    policy-result accept
+                }
+            }
+            statement bgp-underlay {
+                match {
+                    protocol bgp
+                    family [
+                        ipv4-unicast
+                        ipv6-unicast
+                    ]
+                }
+                action {
+                    policy-result accept
+                }
+            }
+            statement bgp-evpn-overlay {
+                match {
+                    family [
+                        evpn
+                    ]
+                }
+                action {
+                    policy-result accept
+                }
+            }
+        }
+        policy leaf-import {
+            default-action {
+                policy-result reject
+            }
+            statement bgp-underlay {
+                match {
+                    protocol bgp
+                    family [
+                        ipv4-unicast
+                        ipv6-unicast
+                    ]
+                }
+                action {
+                    policy-result accept
+                }
+            }
+            statement bgp-evpn-overlay {
+                match {
+                    family [
+                        evpn
+                    ]
+                }
+                action {
+                    policy-result accept
+                }
+            }
+        }
+    }
+--{ running }--[  ]--
+```
+
+///
+
+/// admonition | CLI Ranges
+    type: tip
+Similar to how ranges can be used to pull configuration state from multiple interfaces as an example, in this case a wildcard `*` is used to select multiple routing-policies. The wildcard `spine-*` matches both policies named `spine-import` and `spine-export`.
+///
+
+### Host connectivity and ESI LAG
```

Comment:
from Jorge

"ESI LAG" -> we don't use this in the documentation. I'd call it "LAG Ethernet Segment"

Reply:
added LAG Ethernet Segment but kept ESI LAG as well for familiarity with operators used to other vendors.